stdlib/asyncio/base_events.pyi

```diff
@@ -9,15 +9,28 @@ from asyncio.tasks import Task
 from asyncio.transports import BaseTransport, ReadTransport, SubprocessTransport, WriteTransport
 from collections.abc import Iterable
 from socket import AddressFamily, SocketKind, _Address, _RetAddress, socket
-from typing import IO, Any, Awaitable, Callable, Coroutine, Generator, Sequence, TypeVar, Union, overload
-from typing_extensions import Literal
+from typing import IO, Any, AsyncGenerator, Awaitable, Callable, Coroutine, Generator, Sequence, TypeVar, Union, overload
+from typing_extensions import Literal, TypedDict
 
 if sys.version_info >= (3, 7):
     from contextvars import Context
 
 _T = TypeVar("_T")
 _ProtocolT = TypeVar("_ProtocolT", bound=BaseProtocol)
-_Context = dict[str, Any]
+
+class _BaseContext(TypedDict):
+    message: str
+
+class _Context(_BaseContext, total=False):
```

Comment:
Looking forward to PEP 655.

`default_exception_handler()` also adds a `handle_trackback` field to the context:

https://github.com/python/cpython/blob/0a222db2bca63070f429c0e613707da1bdfaf0e0/Lib/asyncio/base_events.py#L1732-L1733

Reply:
And `Handle._run()` adds `source_traceback` under some conditions:

https://github.com/python/cpython/blob/0a222db2bca63070f429c0e613707da1bdfaf0e0/Lib/asyncio/events.py#L92-L93